tst/div-alg.tst

```diff
@@ -47,7 +47,7 @@ gap> Length(W);
 gap> A:=W[Length(W)];
 [ 1, Rationals, 30, [ [ 2, 11, 0 ], [ 4, 7, 0 ] ], [ [ 15 ] ] ]
 gap> DecomposeCyclotomicAlgebra(A);
-[ [ Rationals, CF(3), [ 1 ] ], [ Rationals, CF(5), [ 0 ] ] ]
+[ [ Rationals, CF(3), [ 1 ] ], [ Rationals, CF(5), [ 9 ] ] ]
```

Comment:
It would be better to glue this change to the particular commit that changed this, but if this is hard, fine to live this as is in a separate commit.